tests/common/broadcom_data.py

```diff
@@ -0,0 +1,5 @@
+SWITCH_HWSKUS = ["Force10-S6000", "Accton-AS7712-32X", "Celestica-DX010-C32",
+                 "Seastone-DX010", "Celestica-E1031-T48S4"]
```

Comment:
We are using many more Broadcom SKU's

Reply:
I know, this is just the list that I copied from `ansible/group_vars/sonic/vars`. It seems like it's been a while since this was updated... 😝

Reply:
in the show version, there is asic field. We should use that instead of the hardware sku, too many skus, too difficult to maintain.

```
Platform: x86_64-juniper_qfx5210-r0
HwSKU: Juniper-QFX5210-64C
ASIC: broadcom
```

Can you later submit another pr to refactor this?